src/parcsr_ls/HYPRE_parcsr_amg.c

```diff
@@ -1859,11 +1859,24 @@ HYPRE_BoomerAMGSetDSLUThreshold (HYPRE_Solver solver,
 
 HYPRE_Int
 HYPRE_BoomerAMGSetCpointsToKeep(HYPRE_Solver solver,
-                                HYPRE_Int  cpt_coarse_level,
-                                HYPRE_Int  num_cpt_coarse,
-                                HYPRE_Int *cpt_coarse_index)
+                                HYPRE_Int    cpt_coarse_level,
+                                HYPRE_Int    num_cpt_coarse,
+                                HYPRE_Int   *cpt_coarse_index)
```

Comment:
@victorapm, should this be HYPRE_BigInt * instead since it is global? Similarly for HYPRE_BoomerAMGSetCPoints( ... ).

Reply:
Yes! Thanks for pointing this out, Daniel. Fixed with 82d208c